packages/next/src/shared/lib/modern-browserslist-target.d.ts

```diff
@@ -1,10 +1,9 @@
 // Keep in sync with the `.js` file.
 declare const MODERN_BROWSERSLIST_TARGET: [
-  'chrome 64',
-  'edge 79',
-  'firefox 67',
-  'opera 51',
-  'safari 12',
+  'chrome 111',
```

Comment:
non-blocking:  why are the literal values in the .d.ts file instead of just something like `declare const MODERN_BROWSERSLIST_TARGET: ReadonlyArray<string>`?

Reply:
That is indeed a good question, I couldn't find a reason, we can clean this up then.